frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -971,4 +971,16 @@ LIMIT 1000000
             }
         },
     })),
+    beforeUnmount(({ cache }) => {
```

Comment:
so let's clear the cache in the unmount of kea

this isn't a slam dunk since if we're leaking references elsewhere this won't necessarily release the array of data for cleanup